.github/actions/kurl-addon-kots-publisher/index.js

```diff
@@ -8,17 +8,34 @@ const addonPackageUrl = getInput('ADDON_PACKAGE_URL');
 const githubToken = getInput('GITHUB_TOKEN');
 const github = getOctokit(githubToken);
 const client = new HttpClient();
+
 const latestKurlVersion = await github.rest.repos.getLatestRelease({
   owner: 'replicatedhq',
   repo: 'kurl'
 });
 const kotsAddonVersions = await client.get('https://kots-kurl-addons-production-1658439274.s3.amazonaws.com/versions.json')
   .then(response => response.readBody())
   .then(response => JSON.parse(response));
-kotsAddonVersions.unshift({
+
+const appendVersion = (kotsAddonVersions, version) => {
+  kotsAddonVersions = kotsAddonVersions.filter(el => el.version !== version.version);
+  kotsAddonVersions.unshift(version);
+  return kotsAddonVersions.sort((a, b) => {
```

Comment:
does this affect the version dropdown?  alpha and nightly are listed at the bottom today. that probably shouldn't change.

Reply:
It shouldnt as those versions are not yet included in this list but are merged with versions embedded in kurl